src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
+    display: flex;
+    margin: 32px 19px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here, but ensure that it is defined somewhere in your SCSS files. If it's not defined, this will cause an error during compilation.